src/main/kotlin/com/ampnet/projectservice/service/OrganizationService.kt

```diff
@@ -18,6 +18,7 @@ interface OrganizationService {
     fun findOrganizationById(organizationUuid: UUID): Organization?
     fun findAllOrganizationsForUser(userUuid: UUID): List<OrganizationWitProjectCountServiceResponse>
     fun findByIdWithMemberships(organizationUuid: UUID): Organization?
+    fun getAllActiveOrganizations(pageable: Pageable, active: Boolean = true): Page<Organization>
```

Comment:
Either rename function to getAllByActive or leave getAllActiveOrganizations but without active param.